flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectTest.java

```diff
@@ -222,6 +225,26 @@ private static void waitForJobFinish(TableResult tableResult) throws Exception {
 		tableResult.getJobClient().get().getJobExecutionResult(Thread.currentThread().getContextClassLoader()).get();
 	}
 
+	@Test
+	public void testFunction() {
+		tableEnv.executeSql(String.format("create function my_abs as '%s'", GenericUDFAbs.class.getName()));
```

Comment:
do we have any test that can verify this created function can really work?

Reply:
I have updated the test to verify that.